src/lib/basecommands/UnauthenticatedExecRenderBaseCommand.tsx

```diff
@@ -0,0 +1,49 @@
+import { FC, ReactNode, Suspense } from "react";
+import { render, Text } from "ink";
+import { RenderContextProvider } from "../../rendering/react/context.js";
+import { UnauthenticatedExtendedBaseCommand } from "./UnauthenticatedExtendedBaseCommand.js";
+import { CoreBaseCommand } from "./CoreBaseCommand.js";
+
+const RenderComponent: FC<{ render: () => ReactNode }> = (p) => (
+  <>{p.render()}</>
+);
+
+function wrapRender<TRes>(
+  fn: (result: TRes) => ReactNode,
+): (result: TRes) => ReactNode {
+  return (result) => {
+    const innerResult = fn(result);
+    if (typeof innerResult === "string") {
+      return <Text>{innerResult}</Text>;
+    }
+
+    return innerResult;
+  };
+}
+
+export abstract class UnauthenticatedExecRenderBaseCommand<
```

Comment:
This class is completely unused atm. Shouldn't logically commands like the `login` subcommands extend this new class?